infra/hooks/postprovision.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
-echo  "Building contosochatapi:latest..."
+echo  "#### Building contosochatapi:latest... ####"
+
 az acr build --subscription ${AZURE_SUBSCRIPTION_ID} --registry ${AZURE_CONTAINER_REGISTRY_NAME} --image contosochatapi:latest ./src/ContosoChatAPI/ContosoChatAPI
-image_name="${AZURE_CONTAINER_REGISTRY_NAME}.azurecr.io/contosochatapi:latest"
-az containerapp update --subscription ${AZURE_SUBSCRIPTION_ID} --name ${SERVICE_ACA_NAME} --resource-group ${RESOURCE_GROUP_NAME} --image ${image_name}
-az containerapp ingress update --subscription ${AZURE_SUBSCRIPTION_ID} --name ${SERVICE_ACA_NAME} --resource-group ${RESOURCE_GROUP_NAME} --target-port 8080
```

Comment:
What is the expectation here? You are building the image but now is the container app getting updated to point to the new image?

Reply:
Since you have added the containerapp configuration in the `azure.yaml` `azd` is now going to do its own deploy on top of this remote build.

Reply:
@wbreza - do you know why this is "read-only"? I can't remove the az scripts.